<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>859</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(65, 72, 51);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="hello_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Palatino Linotype&quot;;
color: rgb(194, 197, 170)</string>
    </property>
    <property name="text">
     <string>Hello </string>
    </property>
   </widget>
   <widget class="QLabel" name="record_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Palatino Linotype&quot;;
color: rgb(194, 197, 170)</string>
    </property>
    <property name="text">
     <string>Record:</string>
    </property>
   </widget>
   <widget class="QLabel" name="coins_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Palatino Linotype&quot;;
color: rgb(194, 197, 170)</string>
    </property>
    <property name="text">
     <string>Coins:</string>
    </property>
   </widget>
   <widget class="QLabel" name="characters_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Palatino Linotype&quot;;
color: rgb(194, 197, 170)</string>
    </property>
    <property name="text">
     <string>Characters:</string>
    </property>
   </widget>
   <widget class="QLabel" name="background_label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Palatino Linotype&quot;;
color: rgb(194, 197, 170)</string>
    </property>
    <property name="text">
     <string>Background:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_Button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>490</y>
      <width>371</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Palatino Linotype&quot;;
background-color: rgb(194, 197, 170);
color: rgb(65, 72, 51)</string>
    </property>
    <property name="text">
     <string>click to start</string>
    </property>
   </widget>
   <widget class="QLabel" name="photo_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="choose_Button">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>450</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Palatino Linotype&quot;;
background-color: rgb(194, 197, 170);
color: rgb(65, 72, 51)</string>
    </property>
    <property name="text">
     <string>save changes</string>
    </property>
   </widget>
   <widget class="QLabel" name="back_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="level_label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Palatino Linotype&quot;;
color: rgb(194, 197, 170)</string>
    </property>
    <property name="text">
     <string>Level:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next_Button">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>130</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Palatino Linotype&quot;;
background-color: rgb(194, 197, 170);
color: rgb(65, 72, 51)</string>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_Button">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Palatino Linotype&quot;;
background-color: rgb(194, 197, 170);
color: rgb(65, 72, 51)</string>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="morecoins_Button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Palatino Linotype&quot;;
background-color: rgb(194, 197, 170);
color: rgb(65, 72, 51)</string>
    </property>
    <property name="text">
     <string>want more coins?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>859</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
